cmd/make/make.go

```diff
@@ -75,15 +75,23 @@ func main() {
 		opts = append(opts, make.WithGithubActionOutput())
 	}
 
-	optsWithCgo := append(opts, make.WithCgo())
+	// We need to avoid cgo on windows. See
+	// https://github.com/golang/go/issues/22439 which still
```

Comment:
This issue was closed > 4 years ago, did you run the repro steps from the issue with a cgo component, or what made you think the issue still relevant/unfixed?

Reply:
I'll flesh in the words in this PR (right now, notes are in a private Kolide ticket) but the fix that issue discusses was only applied to not-CGO. eg: cgo still uses the larger allocations. I think there probably some _other_ issue lurking, but this feels like add on effect.

Reply:
ok gotcha, thanks for clarifying!